torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1,3 +1,4 @@
+import unittest
```

Comment:
fix this lint, if you use `sandcastle_skip_if`, you probably won't need to import unittest